### ISSUE TITLE
runHaxeCode returns nil (0.7h aka latest commit)

### ISSUE THREAD

**Issue by TheLeerName** [ShadowMario/FNF-PsychEngine#12859]
### Describe your bug here.

for some reason runHaxeCode returns `nil`
```lua
luaDebugMode = true
debugPrint(runHaxeCode("return 0"))
```
reporting for latest commit on this moment https://github.com/ShadowMario/FNF-PsychEngine/commit/cfcf124e41dd3f469d31731f7bb1b988f4d79010
i think its issue with SScript

idk here my libs
![cmd_9EVxPbZDvo](https://github.com/ShadowMario/FNF-PsychEngine/assets/85291330/ea94e113-82c2-460b-8cf3-a884aca391ab)

### Command Prompt/Terminal logs (if existing)

```bash
source/backend/Conductor.hx:128: new BPM map BUDDY []
source/psychlua/HScript.hx:18: initializing haxe interp for: mods/scripts/g.lua
source/psychlua/FunkinLua.hx:1564:
source/psychlua/FunkinLua.hx:1435: lua file loaded succesfully:mods/scripts/g.lua
```

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

nah

**Comment by Drawoon2** [comment]
`luaDebugMode = true
debugPrint(runHaxeCode("return 0;"))`

**Comment by TheLeerName** [comment]
> `luaDebugMode = true debugPrint(runHaxeCode("return 0;"))`

nah it not work too

**Comment by ShadowMario** [comment]
You have to put it inside onCreate, this was fixed on 0.7h (not released yet)

**Comment by TheLeerName** [comment]
nah it not work too???????????????????
